src/bundle/Resources/config/services.yaml

```diff
@@ -59,6 +59,8 @@ services:
         lazy: true
         arguments:
             $configResolver: '@ezpublish.config.resolver'
+        calls:
+            - [setLogger, ['@?logger']]
```

Comment:
With proper autowiring/autoconfiguration this method should be called by Symfony framework on it's own.

Consider adding a `monolog.logger` tag with `ibexa.admin` channel.